Package.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let package = Package(
+    name: "MUXSDKStats",
+    platforms: [
+        .iOS(.v9),
+        .tvOS(.v9)
+    ],
+    products: [
+        .library(name: "MUXSDKStats", targets: ["MUXSDKStatsTargets"])
+    ],
+    dependencies: [
+        .package(
+            name: "MuxCore",
+            url: "https://github.com/AndrewBarba/stats-sdk-objc.git",
+            .branch("master")
```

Comment:
This needs to be updated to the correct urls once the core PR is merged and a new release is made